### PR TITLE
refactor plus endgame

### PR DESCRIPTION
**Pull request by nikolaswise**

### CODE REVIEW

**Comment by nikolaswise** [comment]
@paulcpederson

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "js:production": "npm run js -- -m",
     "css": "myth source/css/app.css build/app.css",
     "img": "imagemin-newer source/img/ build",
-    "html": "cp source/*.* build/",
+    "html": "static-site -s source/ -b build/",
+    "posthtml": "cp build/200/index.html build/200.html",
```
Comment (paulcpederson) [inline package.json:12]: ummm, can't you just turn off directory indexes?

Comment (nikolaswise) [inline package.json:12]: lol maybe?

Comment (paulcpederson) [inline package.json:12]: yeah, put `prettyUrl: false` in your front matter

https://github.com/paulcpederson/static-site#front-matter

Comment (nikolaswise) [inline package.json:12]: 👍

**Pull request merged**